stable/pipelines/values.yaml

```diff
@@ -623,7 +648,7 @@ pipelines:
       installerIP: "1.2.3.4"
       installerAuthToken: "{{ .Values.pipelines.authToken }}"
       installerImage: "jfrog/pipelines-installer"
-      registryUrl: "{{ include "pipelines.imageRegistry" . }}"
+      registryUrl: "{{ .Values.global.imageRegistry }}"
```

Comment:
@rahulsadanandan  please check  if this is used anywhere in pipelines installer ?

Reply:
Not sure where this is used , 
@jatindogra @rimusz Can we hardcode this value in systemYaml to `docker.bintray.io` like it is used in 
`languageImages:` section